Dockerfile

```
@@ -28,21 +27,14 @@ RUN adduser \
     --uid "${UID}" \
     "${USER}"
 
-# Speed up build by leveraging docker layer caching
-COPY go.mod go.sum ./
-RUN go mod download
-
-ADD . /work
-RUN make
-
-FROM alpine
+FROM scratch
 LABEL name="Seccomp Operator" \
       version="v0.0.1" \
       description="The Seccomp Operator makes it easier for cluster admins to manage their seccomp profiles and apply them to Kubernetes' workloads."
 
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=build /etc/passwd /etc/passwd
```

We could follow the same logic and make this simpler by amending the files directly:
```suggestion
RUN echo "2000:x:2000:2000::/nonexistent:/sbin/nologin" >> /etc/passwd &&
    echo "2000:x:2000:" >> /etc/group
```

---

See https://github.com/saschagrunert/seccomp-operator/pull/28#issuecomment-651650217